pkg/daemon/ceph/client/command.go

```diff
@@ -41,7 +41,7 @@ const (
 	// CrushTool is the name of the CLI tool for 'crushtool'
 	CrushTool             = "crushtool"
 	CmdExecuteTimeout     = 1 * time.Minute
-	cephConnectionTimeout = "15" // in seconds
+	cephConnectionTimeout = "600" // in seconds
```

Comment:
When the operator runs `ceph` commands, why does the timeout have to be long or match the auth timeout? Since the timeout was changed to 15s a while back I've seen this as a really good thing because in the common case of failure there was just some intermittent issue and the operator can try again after 15s. If the timeout is long, there are several side effects such as being a long time until the operator retries, the operator appearing hung, etc. We don't want the operator to get stuck for long periods or else it can't continue to manage the cluster.

Reply:
@jdurgin based on the above comment, do you still think we should pursue that? (relates to https://bugzilla.redhat.com/show_bug.cgi?id=1767390).

Thanks!

Reply:
Thanks for the explanation @travisn, that makes sense. If rook is doing the retries rather than the ceph cli's internals, that's fine, and I agree there isn't a need to change this timeout.